.github/workflows/deploy-production.yml

```diff
@@ -10,7 +10,9 @@ on:
         required: true
 
 env:
-  REACT_APP_API_URL: ${{ secrets.REACT_APP_API_URL }}
+  REACT_APP_API_BASE_URL: ${{ secrets.REACT_APP_API_BASE_URL }}
+  REACT_APP_API_CLIENT_ID: ${{ secrets.REACT_APP_API_CLIENT_ID }}
+  REACT_APP_API_CLIENT_SECRET: ${{ secrets.REACT_APP_API_CLIENT_SECRET }}
```

Comment:
Do we need to set these variables in these workflows (deploy_xxx)?
Should it work without them?

Reply:
No, we don't I removed it from every workflow except for test workflow already 👍